packages/plotly/plotly.1.0.0/opam

```diff
@@ -0,0 +1,42 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Binding for Plotly Open Source Graphing Library"
+description:
+  "OCaml wrapper for Plotly Open Source Graphing Library: https://plotly.com/graphing-libraries"
+maintainer: ["Jun FURUSE <jun.furuse@dailambda.jp>"]
+authors: ["Jun FURUSE <jun.furuse@dailambda.jp>"]
+license: "MIT"
+homepage: "https://github.com/camlpsotter/plotly-ocaml"
+bug-reports: "https://github.com/camlpsotter/plotly-ocaml/issues"
+depends: [
+  "ocaml"
+  "dune" {>= "3.7"}
+  "js_of_ocaml"
+  "js_of_ocaml-ppx"
+  "js_of_ocaml-lwt"
+  "pyml"
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/camlpsotter/plotly-ocaml.git"
```

Comment:
```suggestion
dev-repo: "git+https://github.com/camlspotter/plotly-ocaml.git"
```

Reply:
Ouch.  Thank you. I will fix `dune-project` and resubmit.